src/objectSorter.ts

```diff
@@ -276,6 +276,9 @@ function objectSorter(
   stringifiers.array = sortOptions.array
     ? (str._arraySort.bind(stringifiers) as objectSorter.StringifyFn)
     : (str._array.bind(stringifiers) as objectSorter.StringifyFn);
+  stringifiers.typedarray = sortOptions.array
+    ? (str._typedArraySort.bind(stringifiers) as objectSorter.StringifyFn)
+    : (str._typedArray.bind(stringifiers) as objectSorter.StringifyFn);
```

Comment:
I think we need a new option for this case, since in buffers and typed arrays order of bytes determine what's inside.

And I think it should be disabled by default, event if shorthand option `sort: true` is enabled. So the only way to enable typed array sorting should be enabling it explicitly via
```
sortOptions.typedArray
```